app/grandchallenge/algorithms/static/algorithms/js/job_session.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
```

Comment:
Completely unrelated, but I wonder if we should only create js modules from now on? Maybe on for @HarmvZ